api/app/Policies/ClassificationPolicy.php

```diff
@@ -41,7 +41,7 @@ public function view(User $user = null)
      */
     public function create(User $user)
     {
-        return $user->isAdmin();
+        return $user && $user->isAbleTo("create-any-classification");
```

Comment:
Not a big deal but PHP 8 has null safe method operator. https://stitcher.io/blog/php-8-nullsafe-operator  Since this is the first PR of the set it will probably be copied for all the rest.

Reply:
Which method do we prefer? Should I update to setting `null` by default or use the method documented?

Reply:
okay, it has been updated to use the documented method.

[update to allow guest users](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5801/commits/3dbc9e7d6d7edd0df4099987aedcc19995b5bb99)